### PR TITLE
fix waddrmgr tests

### PR DESCRIPTION
**Pull request by jolan**
Mostly a conversion of Bitcoin -> Decred addresses and removing some things that don't make sense for Decred.

### CODE REVIEW

File: wallet/createtx_test.go [hunk 1]
```diff
@@ -0,0 +1,253 @@
+package wallet
```
Comment (jcvernaleo) [inline wallet/createtx_test.go]: Where did this file come from?
It is not in dcrwallet master as far as I can tell.  And it doesn't compile.

File: waddrmgr/sync.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ const (
 	// maxRecentHashes is the maximum number of hashes to keep in history
 	// for the purposes of rollbacks. Note that if reorganizations longer
 	// than this occur, corruption of the wallet may occur.
-	maxRecentHashes = 1000
+	maxRecentHashes = 20
```
Comment (cjepson) [inline waddrmgr/sync.go]: Whoa, that should not be changed. Leave it at 1000.

**Comment by jolan** [comment]
There wasn't enough test data for maxRecentHashes to be 1000 which is why I ratcheted it down.  I've added more blocks for it to work and put it back to 1000.

Also removed that errant file that @jcvernaleo mentioned.

**Comment by jcvernaleo** [comment]
Looks good now.  Just rebase and squash then OK.

**Comment by alexlyp** [comment]
looks good

**Pull request merged**